.github/renovate.json

```diff
@@ -1,11 +1,34 @@
 {
   "extends": [
     "config:base",
-    "schedule:weekdays",
-    ":preserveSemverRanges"
+    "schedule:weekly",
+    ":automergeLinters",
+    ":automergeMinor",
+    ":automergeTesters",
+    ":enableVulnerabilityAlerts",
+    ":rebaseStalePrs",
+    ":semanticCommits",
+    ":updateNotScheduled"
   ],
-  "prConcurrentLimit": 5,
-  "includePaths": [
-    "package.json"
-  ]
```

Comment:
I wonder if this is needed to prevent renovate from looking at python stuff. https://docs.renovatebot.com/configuration-options/#includepaths

It might be worth considering adding `enabledManagers`
> If you are more interested in including only certain package managers (e.g. npm), then consider enabledManagers instead.

but I'm also not sure if any of the configs this is extending already have that.

Reply:
Yea, good call, let's just be explicit that we care about the npm requirements for this.